xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4875,6 +4875,7 @@ void CVideoPlayer::OnLostDisplay()
   m_VideoPlayerVideo->SendMessage(new CDVDMsgBool(CDVDMsg::GENERAL_PAUSE, true), 1);
   m_clock.Pause(true);
   m_displayLost = true;
+  FlushRenderer();
```

Comment:
This string ` FlushRenderer();`  causes video freeze on 4-8 seconds on amlogic devices if display changes refresh rate during video start.

Reply:
why don't you create a ticket for amlogic related issues?